programs/drift/src/controller/liquidation.rs

```diff
@@ -1888,6 +1910,18 @@ pub fn liquidate_perp_pnl_for_deposit(
         return Err(ErrorCode::InvalidLiquidation);
     }
 
+    if safest_tier_perp_liability < contract_tier
```

Comment:
on ContractTier and AssetTier, might be nice to implement a `safest` and `is_less_safe` method so this is easier to understand